src/main/java/edu/kis/powp/jobs2d/drivers/DriverManager.java

```diff
@@ -1,27 +1,25 @@
 package edu.kis.powp.jobs2d.drivers;
 
 import edu.kis.powp.jobs2d.Job2dDriver;
-import edu.kis.powp.jobs2d.LoggerDriver;
 
 /**
  * Driver manager provides means to setup the driver. It also enables other
  * components and features of the application to react on configuration changes.
  */
 public class DriverManager {
 
-	private Job2dDriver currentDriver = new LoggerDriver();
+	private final MacroProxyDriver currentDriver = new MacroProxyDriver();
```

Comment:
Try avoiding enforcing MacroProxyDriver in driver manager (an unwanted coupling).